.github/workflows/system-tests.yml

```diff
@@ -71,31 +82,11 @@ jobs:
           path: artifact.tar.gz
 
   parametric:
-    runs-on: ubuntu-latest
-    env:
-      TEST_LIBRARY: nodejs
-    steps:
-      - name: Checkout system tests
-        uses: actions/checkout@v4
-        with:
-          repository: 'DataDog/system-tests'
-      - uses: actions/setup-python@v4
-        with:
-          python-version: '3.9'
-      - name: Checkout dd-trace-js
-        uses: actions/checkout@v4
-        with:
-          path: 'binaries/dd-trace-js'
-      - name: Build
-        run: ./build.sh -i runner
-      - name: Run
-        run: ./run.sh PARAMETRIC
-      - name: Compress artifact
-        if: ${{ always() }}
-        run: tar -czvf artifact.tar.gz $(ls | grep logs)
-      - name: Upload artifact
-        uses: actions/upload-artifact@v3
-        if: ${{ always() }}
-        with:
-          name: logs_parametric
-          path: artifact.tar.gz
+    needs:
+      - build-artifacts
+    uses:  DataDog/system-tests/.github/workflows/system-tests.yml@main
```

Comment:
How to pass additional arguments to the runner with this reusable workflow? For example `--splits=8 --group=4`.

Reply:
I'll add them in your PR